agent/src/agent/pipeline/config/jython_scripts/sage.py

```diff
@@ -88,20 +90,7 @@ def main():
                         sdc.log.debug(str(res.json()))
                         break
                     except requests.HTTPError as e:
-                        event = sdc.createEvent('sage_error', 1)
-                        event.value = {
-                            'value': 1,
-                            'properties': {
-                                'what': 'sage_http_error',
-                                'target_type': 'counter',
-                                'code': e.response.status_code,
-                                'pipeline_name': sdc.userParams['PIPELINE_NAME']
-                            },
-                            'timestamp': time.time()
-                        }
-                        cur_batch.addEvent(event)
-                        cur_batch.process(entityName, offset)
-                        cur_batch = sdc.createBatch()
+                        requests.post(sdc.userParams['MONITORING_URL'] + str(e.response.status_code))
```

Comment:
since we don't produce events anymore, you can remove cur_batch_.eventCount() from line 141, remove events production from this stage and remove the unneeded stages from the pipeline that were called to process events